app/views/AdminView/components/Users/components/AddEditUserDialog/index.tsx

```diff
@@ -456,6 +457,7 @@ const AddEditUserDialog = ({
             <Checkbox checked={gkbAdd} onChange={handleCheckboxChange} />
           }
           disabled={Boolean(editData)}
+          sx={{ paddingLeft: '6px' }}
```

Comment:
was adding this to add-user__checkbox not enough that warrants an in-line instead? 

If possible we'd like to do it in className instead since it's already setup, as future devs might just edit that one instead and this in-line here might cause hiccups

Reply:
yea you're right I'm changing it